README.md

```diff
@@ -262,6 +262,19 @@ Documentation on currently available API is provided in [PSPDFKit.js](https://gi
 
 ## Troubleshooting
 
+### Error Reporting
+To get a proper error reporting you need to register a global error listener which will print most of runtime errors into Logcat.
+Put the following code snippet into `[your-project]/platforms⁩/android⁩/app⁩/src⁩/main⁩/assets⁩/⁨www⁩/js⁩/index.js`:
```

Comment:
This is on a new line. Do you want to have paragraph here? Can you add empty line before?